packages/connect/src/__tests__/debug.tests.js

```diff
@@ -143,31 +143,6 @@ describe("debugger", () => {
     ]);
   });
 
-  it("should debug clear operations", () => {
```

Comment:
Removed this test because it tests a built-in object `Map` which we do not support anymore

Reply:
👍